trade-finance-manager-ui/templates/_partials/primary-navigation.njk

```diff
@@ -29,10 +29,14 @@
 
 {% set navItems = [] %}
 
-{% if user and user | userIsInTeam(['PDC_READ', 'PDC_RECONCILE']) %}
-  {% set navItems = [allDealsNavItem, allFacilitiesNavItem, bankReportsNavItem] %}
-{% elif user %}
-  {% set navItems = [allDealsNavItem, allFacilitiesNavItem] %}
+{% if user %}
+  {% if user | userIsOnlyInTeam(['PDC_READ']) or user | userIsOnlyInTeam(['PDC_RECONCILE']) %}
+    {% set navItems = [bankReportsNavItem] %}
+  {% elif user | userIsInTeam(['PDC_READ', 'PDC_RECONCILE']) %}
+    {% set navItems = [allDealsNavItem, allFacilitiesNavItem, bankReportsNavItem] %}
+  {% else %}
+    {% set navItems = [allDealsNavItem, allFacilitiesNavItem] %}
+  {% endif %}
```

Comment:
I'm not a huge fan of all this nesting as it is fairly complicated, but I can't think of a good way of shortening this in pure nunjucks. Any suggestions are appreciated

Reply:
I think this is fine - it's not nesting too deeply, and our helper functions make it quite readable.

What would happen in the case where the user has both the `PDC_READ` and `PDC_RECONCILE` permissions but nothing else? I think this would fall through to the second case where they view everything again? I don't think we would expect this but should probably account for it.

Reply:
I think although it's quite long it wasn't too complex to understand

Reply:
@toddym42 That's a good point. Just to capture that edge case, I added an extra
```